includes/pg-functions.php

```diff
@@ -158,13 +161,8 @@ function updateThousandsSeparatorDbConfig()
     }
     $tableName         = $wpdb->prefix . PG_CONFIG_TABLE_NAME;
     $thousandSeparator = get_option('woocommerce_price_thousand_sep');
-    $wpdb->update(
-        $tableName,
-        array('value' => $thousandSeparator),
-        array('config' => 'PAGANTIS_SIMULATOR_THOUSANDS_SEPARATOR'),
-        array('%s'),
-        array('%s')
-    );
+    $wpdb->update($tableName, array('value' => $thousandSeparator), array('config' => 'PAGANTIS_SIMULATOR_THOUSANDS_SEPARATOR'),
```

Comment:
Once you use multiline to call a funciton, each parameter have to be in it's own line. 
https://www.php-fig.org/psr/psr-2/#44-method-arguments